src/mumble/Settings.h

```diff
@@ -306,6 +306,7 @@ struct Settings {
 	/// socket to the same address as its TCP
 	/// connection is using.
 	bool bUdpForceTcpAddr;
+	bool bHideOS;
```

Comment:
Please move this down below `QString qsSslCiphers;`
Add a comment above it:

    // Privacy settings